README.adoc

```diff
@@ -44,7 +44,7 @@ The operator is now ready to create Jaeger instances!
 
 === OpenShift
 
-The instructions from the previous section also work on OpenShift given that the `operator-openshift.yaml` is used instead of `operator.yaml`. Make sure to install the RBAC rules, the CRD and the operator as a privileged user, such as `system:admin`.
+The instructions from the previous section also work on OpenShift. Make sure to install the RBAC rules, the CRD and the operator as a privileged user, such as `system:admin`.
```

Comment:
Could be the whole section removed? Instead have just one deploy section. Less text to maintain and allows us  catching differences between platforms sooner.

Reply:
You mean, the whole OpenShift section?

Reply:
yes

Reply:
I'll open an issue to discuss it. I think it still has value, especially about logging as admin to create the objects that can't be created with an ordinary user.

Reply:
There can be a sentence saying log as admin and use oc instateak of kubectl

Reply:
As far as I know, `kubectl` can be used instead of `oc` for OpenShift as it will cleanly translate to the appropriate commands. I opened an issue (#222) to discuss this further. I'm OK in removing it if the Kubernetes steps can be followed for OpenShift as well.